nominees/berkely-air-sums.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Berkely Air Sums",
+    "description": "The Stove Use Monitoring System (SUMS) provides insights into cookstove usage patterns, number of meals cooked, and time of use by recording stove temperature changes. Data sampled by SUMS are uploaded to a computer via a data cable and further analyzed using proprietary algorithms developed to quantify cookstove usage in households.",
+    "website": "https://berkeleyair.com/",
+    "license": [
+      {
+        "spdx": "",
```

Comment:
Projects must demonstrate the use of an approved open license. @ijeomaemeruwa this is a mandatory field and cannot be merged without the an open source license indicated. Thanks.

Reply:
If a project doesn't have an approved license do i remove it?

Reply:
Yes, correct.

The rationale being that an open source license is required for a project to be nominated to be a digital public good. So if it does meet this minimum requirement, it cannot be nominated.